aws-cli/bash-linux/dynamodb/tests/test_dynamodb_examples.sh

```diff
@@ -127,6 +130,7 @@ function main() {
     "dynamodb_describe_table -n $test_table_name " \
     0
 
+   # shellcheck disable=SC2154
```

Comment:
Looks like shellcheck can't see that `test_command_response` got set globally in `run_test`? Which implies to me that suppressing the SC1091 above is also not an appropriate action.

Reply:
I removed all these suppress statements